monitoring.yml

```diff
@@ -150,7 +153,7 @@ properties:
       - database_type: "influxdb"
         database_name: (( meta.influxdb.database ))
         url: (( "http://" jobs.influxdb.networks.monitoring.static_ips.[0] ":8086" ))
-        name: "influxdb_production"
+        name: (( meta.grafana.name ))
```

Comment:
Yeah, this seems ripe for refactoring. There is also `riemann.influxdb_user`, etc as well..